### ISSUE TITLE
Feature request: Split up menu item body and page body

### ISSUE THREAD

**Issue by guywillis** [adaptlearning/adapt_framework#2391]
Break apart the content object menu item body and page body JSON data fields. The benefit of this is so you can strictly define what text renders in each area. 

Suggestion:
```
"title": "Content object title",
"bodyMenu": "Menu body text",
"bodyPage": "Page body text",
```

Alternatively:
```
"_menu": {
  "itemTitle": "",
}

**Comment by chucklorenz** [comment]
Yes! and would like to have the separate titles regardless of the how the JSON is formed. (But the second is closer to my thought processes when coding properties.schema. Might get an AAT specialist to weigh in. Maybe "title" and "body" are all that is needed if they are grouped into separate "_menu" and "_page" objects.)

**Comment by oliverfoster** [comment]
Duplicate of #2932
